### PR TITLE
Fixes layout issues with filters on sheer pages

### PR DESCRIPTION
**Pull request by jimmynotjim**
Fixes layout issues with filters on sheer pages

## Additions

- Adds sheer only JS

## Removals

- Removes old styles

## Changes

- Updates filters to use atomic expandable
- Updates filters to use atomic filterable-list-controls
- Updates un-filtered lists to include total posts

## Testing

- `gulp build` and navigate to `/blog`, `/newsroom`, `/activity-log`, and `/the-bureau/leadership-calendar/`
- Expandable should open/close
- Notification should always show total posts
- Layout should be consistent w/ Wagtail page filters

## Review

- @schaferjh 
- @sebworks 
- @anselmbradford 
- @KimberlyMunoz 

## Screenshots

screen shot 2016-03-16 at 2 48 07 pm

screen shot 2016-03-16 at 2 48 43 pm

screen shot 2016-03-16 at 2 49 02 pm

screen shot 2016-03-16 at 2 49 16 pm

## Notes

- There's more removing of old sheer styles we could probably do, but I tried to limit the scope of this PR

## Checklist

* [ ] Changes are limited to a single goal (no scope creep)
* [ ] Code can be automatically merged (no conflicts)
* [ ] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [ ] Passes all existing automated tests
* [ ] New functions include new tests
* [ ] New functions are documented (with a description, list of inputs, and expected output)
* [ ] Placeholder code is flagged
* [ ] Visually tested in supported browsers and devices
* [ ] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)

### CODE REVIEW

File: cfgov/jinja2/v1/_includes/macros/filters.html [hunk 1]
```diff
@@ -185,172 +191,175 @@
...
-                             'Raj Date',
-                             'Elizabeth Warren'] %}
+        {% endif %}
+
+        {# Calendar filters #}
+        {% if filters.calendar %}
+        {% set directors_list = [
+            'Richard Cordray',
+            'David Silberman',
+            'Meredith Fuchs',
```
Comment (sebworks) [inline cfgov/jinja2/v1/_includes/macros/filters.html]: Shouls `Fuchs` still be in here?

Comment (jimmynotjim) [inline cfgov/jinja2/v1/_includes/macros/filters.html]: Yep, same as all the other past Dept Directors.

Comment (sebworks) [inline cfgov/jinja2/v1/_includes/macros/filters.html]: :+1:

**Comment by schaferjh** [comment]
Nice.

**Comment by KimberlyMunoz** [comment]
This might be a pre-coffee question, but why is the label `From: (YYYY-MM)` and the placeholder `MM-DD-YYYY`?

File: cfgov/unprocessed/js/routes/sheer.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+var Expandable = require( '../molecules/Expandable' );
+
+var filterableListDom = document.querySelector( '.o-filterable-list-controls' );
+var filterableListControls;
+for ( var i = 0, len = filterableListDom.length; i < len; i++ ) {
+  filterableListControls = new FilterableListControls( document.body )
+  filterableListControls.init();
+}
+
+var expandableDom = document.querySelectorAll( '.m-expandable' );
```
Comment (anselmbradford) [inline cfgov/unprocessed/js/routes/sheer.js]: Just confirming, this never appears on a page that contains an ExpandableGroup?

File: cfgov/jinja2/v1/_layouts/base.html [hunk 3]
```diff
@@ -206,6 +206,12 @@
     {% include '/js/routes/common.js' %}
 </script>
 
+{% if not page %}
+<script type='text/javascript'>
```
Comment (anselmbradford) [inline cfgov/jinja2/v1/_layouts/base.html]: Add `TODO` note to remove when Sheer pages are removed.

File: cfgov/unprocessed/js/routes/sheer.js (hunk 2 above)
Comment (jimmynotjim) [inline cfgov/unprocessed/js/routes/sheer.js]: That's one thing I wasn't sure about. Are there any sheer pages using the atomic expandable group? Maybe the structure page? @sebworks do you know?

**Comment by jimmynotjim** [comment]
> This might be a pre-coffee question, but why is the label From: (YYYY-MM) and the placeholder MM-DD-YYYY?

@KimberlyMunoz Is there JS changing them, because the label in the template matches https://github.com/cfpb/cfgov-refresh/pull/1598/files#diff-30cc34a83351df1ad79fc3b6e4d65a13R318

File: cfgov/unprocessed/js/routes/sheer.js (hunk 2 above)
Comment (sebworks) [inline cfgov/unprocessed/js/routes/sheer.js]: There shouldn't be any pages. There are pages that use the `atomic expandable` but not the `atomic-expandable-group`.

**Comment by sebworks** [comment]
>@KimberlyMunoz Is there JS changing them, because the label in the template matches https://github.com/cfpb/cfgov-refresh/pull/1598/files#diff-30cc34a83351df1ad79fc3b6e4d65a13R318

Yep, so we just set them to `(YYYY-MM)` in case JS was disabled. That is the format that sheer is expecting.  https://github.com/cfpb/cfgov-refresh/blob/flapjack/cfgov/sheerlike/filters.py#L72.

**Comment by jimmynotjim** [comment]
@sebworks ah ok, so we need to update the label as well as the placeholder when the script is initialized?

**Comment by KimberlyMunoz** [comment]
TIL. Thanks for the explanation. I think it would make sense to update the label as well so users aren't getting conflicting information about what format to enter the date.

**Comment by jimmynotjim** [comment]
Ready for sign-off.

**Comment by sebworks** [comment]
Yeah, it was supposed to do that `post-filter.js` but it looks like it isn't working.

**Comment by sebworks** [comment]
:+1:

**Comment by KimberlyMunoz** [comment]
:+1:

**Comment by jimmynotjim** [comment]
Just realized this has a secondary effect of fixing the side-nav on sheer pages. Check `/the-bureau`and  `/the-bureau/bureau-structure` in small views and make sure the nav open/closes and the structure slides back and forth.

**Comment by KimberlyMunoz** [comment]
:+1:

**Pull request merged**